Tools/autotest/param_metadata/htmlemit.py

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env python
+"""
+Emit docs in a form acceptable to the Ardupilot wordpress docs site
```

Comment:
Change _Ardupilot_ to _old ArduPilot_

In the future we should change this to generate a normal html file.